.coveralls.yml

```diff
@@ -1,2 +0,0 @@
-service_name: travis-pro
-repo_token: eoFbGKxivT1jjyPsUJOuzvrTDDrtCLQQw
```

Comment:
Odd; I put .coveralls.yml into .gitignore so it wouldn't overwrite the master branch copy! Sorry about that...